docs/configuration.md

```diff
@@ -0,0 +1,51 @@
+# Environment Configuration
+
+When developing a Probot App, you will need to have several different fields in a `.env` file which specify environment variables. Here are some common use cases:
+
+```
```

Comment:
The big blob of preformatted text is a little hard to read. What would you think about making it into a table?

Variable | Description
---|---
`WEBHOOK_PROXY_URL` | This is used to allow smee to communicate as your Webhook URL
`WEBHOOK_SECRET` | This is the webhook secret used when creating a GitHub App
`APP_ID` | This is the App ID assigned to your GitHub App
`PRIVATE_KEY` | This is your private key; however, this variable is optional, if it is not present Probot will look in your project's directory for the private key.
`PRIVATE_KEY_PATH` | This is the path to your private key.